tomography_preprocessing/denoising/care4relion/care4relion.py

```diff
@@ -0,0 +1,149 @@
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+import starfile
+import rich
+import typer
+import subprocess
+from rich.progress import track
+
+from .train_neural_network import train_neural_network
+from ._utils import *
+from .constants import predict_config_prefix, even_suffix 
+from .._cli import cli
+from ...utils.relion import relion_pipeline_job
+
+console = rich.console.Console(record=True)
+
+@cli.command(name='Care4Relion')
+@relion_pipeline_job
+def care4relion(
+    tilt_series_star_file: Path = typer.Option(...),
+    output_directory: Path = typer.Option(...),
+    training_tomograms: Optional[str] = typer.Option(None),
+    model_name: Optional[Path] = typer.Option(None),
+    n_tiles: Optional[Tuple[int,int,int]] = typer.Option((1,1,1))
+
+):
+    """Denoises tomograms using cryoCARE (Thorsten Wagner version) (https://github.com/thorstenwagner/cryoCARE_mpido)
+    
+    Requires that two tomograms have been generated using the same sample. These can be generated via taking odd/even 
+    frames during Motion Correction (optimal) or by taking odd/even tilts during tomogram reconstruction.
+    The location of these tomograms should be specified in the global star file for all tilt series with the headers: 
+    
+     
+    _rlnTomoTomogramHalvesForDenoisingEven
+    
+      _rlnTomoTomogramHalvesForDenoisingOdd
```

Comment:
really not a fan of these names, how about

`rlnTomoTomogramHalf1` and `rlnTomoTomogramHalf2` to match postprocess?

Reply:
Me neither, in rln I went for rlnTomoReconstructedEvenTomogram which matches the tomogram generation label rlnTomoReconstructedTomogram

Reply:
Nice! Slight preference for rlnTomoReconstructedTomogramHalf1, I don't think even/odd is as good a name generally - is that okay with you?

Reply:
I've gone for Odd/Even as that's the MotionCor convention when using splitsum (adds _EVN or _ODD to file names), even/odd may make it easier to trace back to there? Am ok with either approach really

Reply:
Ohh yes, forgot about the alternating frames